deploy/compose/.env

```diff
@@ -13,7 +13,7 @@ COLLECTORIST_API_PORT=8180
 COLLECTOR_OSV_API_PORT=8181
 COLLECTOR_SNYK_API_PORT=8182
 COLLECTOR_NVD_API_PORT=8183
-GUAC_IMAGE=ghcr.io/trustification/guac:v0.3.13-temp
+GUAC_IMAGE=quay.io/mrizzi/guac:20231107-111157-arm64
```

Comment:
`amd64` images
```suggestion
GUAC_IMAGE=quay.io/mrizzi/guac:20231107-134115-amd64
```

Reply:
We now have an "official" image that solves this.